tests/test_asyn_cifar10.py

```diff
@@ -0,0 +1,259 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+import unittest
+
+from federatedscope.core.auxiliaries.data_builder import get_data
+from federatedscope.core.auxiliaries.utils import setup_seed, update_logger
+from federatedscope.core.configs.config import global_cfg
+from federatedscope.core.fed_runner import FedRunner
+from federatedscope.core.auxiliaries.worker_builder import get_server_cls, get_client_cls
+
+
+class AsynCIFAR10Test(unittest.TestCase):
```

Comment:
If possible, consider adding a simply constructed asynchronous example (as shown in Figure 2 of the paper), that asserts that the order of printed client ids is consistent with the order maintained by the simulator of the new PR.